pkg/system/syscall_windows.go

```diff
@@ -61,12 +62,7 @@ var (
 
 // OSVersion is a wrapper for Windows version information
 // https://msdn.microsoft.com/en-us/library/windows/desktop/ms724439(v=vs.85).aspx
-type OSVersion struct {
-	Version      uint32
-	MajorVersion uint8
-	MinorVersion uint8
-	Build        uint16
-}
+type OSVersion osversion.OSVersion
```

Comment:
This could possibly be a type alias

Reply:
ah, hm. good point, forgot about that as an option

I'll do a quick touch-up follow-up

Reply:
https://github.com/moby/moby/pull/40135